src/geo/transform.js

```diff
@@ -380,7 +380,12 @@ class Transform {
      * @param {TileCoord} tileCoord
      * @param {number} maxZoom maximum source zoom to account for overscaling
      */
+
     calculatePosMatrix(tileCoord: TileCoord, maxZoom: number) {
+        return new Float32Array(this._calculatePosMatrix(tileCoord, maxZoom));
+    }
+
+    _calculatePosMatrix(tileCoord: TileCoord, maxZoom: number) {
```

Comment:
😳 That's vestigial from earlier attempts to figure out the right way to calculate tile distance, when I thought a separate helper function was necessary. I just removed the helper, and `cameraToTileDistance` now calls `calculatePosMatrix` directly.